### ISSUE TITLE
/stash/publish missing parameters

### ISSUE THREAD

**Issue by Derewen** [wix-incubator/DeviantArt-API#219]
/stash/publish endpoint is missing few important parameters like
1) subscriber only publish to tiers (I am also not sure there is fetch endpoint for the tiers)
2) image was created with AI
3) inclusion in third party AI datasets

and few others present in the UI

**Comment by tuttle** [comment]
Yes, these three API features are significantly missing. I believe they are not too hard to add, as the UI, database and engine seem to use them. May we ask for them in the API as well?
